src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -241,7 +252,7 @@
     /**
      * Recreated from {@link #owner}
      */
-    /*package*/ transient /*almos final*/ FlowNodeStorage storage;
+    /*package*/ transient /*almos final*/ TimingFlowNodeStorage storage;
```

Comment:
🐛  For forcing TimingFlowNodeStorage in all cases -- extra overheads for the hot path. Recommend we use a quick check where doing timing and only perform those operations if we are tracking timing.